Week-7/Assignment7.6.js

```diff
@@ -0,0 +1,56 @@
+class Queue{
+	
+constructor()
+{
+	this.s1 = [];
+	this.s2 = [];
+}
+
+enQueue(x)
+{
+	
+	// Move all elements from s1 to s2
+	while (this.s1.length != 0)
+	{
+		this.s2.push(this.s1.pop());
+		//s1.pop();
+	}
+
+	// Push item into s1
+	this.s1.push(x);
+
+	// Push everything back to s1
+	while (this.s2.length != 0)
+	{
+		this.s1.push(this.s2.pop());
+		//s2.pop();
+	}
+}
+
+
+deQueue()
+{
+	
+
+	if (this.s1.length == 0)
+	{
+		document.write("Q is Empty");
+	}
+
+
+	let x = this.s1[this.s1.length - 1];
+	this.s1.pop();
+	return x;
+}
+}
```

Comment:
And one general comment just dont copy paste from geeks for geeks try to attempt the code by urself first and if stuck please refer internet.. it will be helpful for ur learning

Reply:
yaa,sure.. actually i was taking help from geeks for geeks